src/plone/restapi/services/configure.zcml

```diff
@@ -27,5 +27,6 @@
   <include package=".workflow"/>
   <include package=".multilingual"
       zcml:condition="have plone-5"/>
-
+  <include package=".contact"
```

Comment:
@sneridagh I think we should be consistent here and call the file "email_notification" instead of "contact".

Reply:
Done.